spec/controllers/catalog_controller_spec.rb

```diff
@@ -1291,16 +1290,18 @@
   end
 
   context "hiding columns" do
-    let(:user) { FactoryBot.create(:user) }
-
     before do
       Tenant.seed
       EvmSpecHelper.local_miq_server
     end
 
     let!(:service_template) { FactoryBot.create(:service_template, :description => 'XXX', :tenant => Tenant.root_tenant) }
```

Comment:
If you rename this to `record` (assuming it's not used elsewhere on this level) in each variable and in other examples, we can clean up the spec even more. Please do it and in the next review iteration I'll add the comments.